utils/check_code_style.py

```diff
@@ -71,7 +71,9 @@
 	# Matches any tabulator characters.
 	"\t": "tabulators should only be used for indentation",
 	# Matches any commas that are not followed by whitespace characters.
-	",\\S": "commas should be followed by whitespaces"
+	",\\S": "commas should be followed by whitespaces",
+	# Matches incorrect structured bindings
+	"&\\s+\\[": "structured bindings should not contain a space after the ampersand: &[first, second]",
```

Comment:
This ends up looking like:
```
ERROR: line 318: structured bindings should not contain a space after the ampersand: &[first, second] in 'for(const auto& [key, s] : loaded)'
```
Which I think is a little confusing. I think the text is clear enough on its own so we should drop the example.
```suggestion
	"&\\s+\\[": "structured bindings should not contain whitespace after the ampersand",
```
Also, change "a space" to "whitespace".